src/Sylius/Bundle/CoreBundle/Validator/Constraints/AtLeastOneOf.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\Constraints\Composite;
+
+/**
+ * @internal
```

Comment:
I would definitely explain here (and in the validator) that these are services copied from Symfony, so we can use them while still supporting Symfony 4.4. Let's also make it explicit that they would be removed when dropping Symfony 4.4 support. This information is IMO crucial to merge it and avoid any confusion :) 🖖